examples/tests/visual.test.ts

```diff
@@ -0,0 +1,45 @@
+// visual test for github.com
+
+import { shortest } from "@antiwork/shortest";
+
+// GitHub header visual test
+shortest("Visit github.com and verify the global navigation header layout. Check GitHub logo, search bar, navigation items (Pull requests, Issues, Marketplace, Explore), and profile dropdown maintain correct spacing and alignment", {
+  url: "https://github.com",
+  regions: ["header.Header-old", ".header-search-button", ".AppHeader-globalBar"],
+  tolerance: 0.1
```

Comment:
Passing tolerance allows for 0.1% pixel difference between baseline and current screenshots. Useful because small visual differences can be acceptable.